### PR TITLE
OpenROAD flow variables document

### PR DESCRIPTION
**Pull request by vijayank88**
Signed-off-by: vijayank88 <paruthi143@gmail.com>

OpenROAD flow public platforms environment variables document contributed.

### CODE REVIEW

File: docs/contrib/PlatformBringUp.md
Comment (vvbandeira) [inline docs/contrib/PlatformBringUp.md:146]: remove space between `] (`

File: docs/user/GettingStarted.md
Comment (vvbandeira) [inline docs/user/GettingStarted.md:102]: remove space between `] (`

File: docs/toc.yml
Comment (vvbandeira) [inline docs/toc.yml:151]: Move to line 147, before Metrics.

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Comment by vvbandeira** [comment]
Looks good now @vijayank88. I will approve and merge once the `=` -> `?=` PR is merged in ORFS.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:54]: `BLACKBOX_MAP_TCL` is deprecated. There shouldn't be any platforms using this.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:123]:
```suggestion
| `PLACE_SITE`          | Placement site for core cells defined in the technology LEF file.                                                                                                                |

```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:225]:
```suggestion
| `CORE_UTILIZATION`    | The core utilization percentage (0-100). Overrides DIE_AREA & CORE_AREA.                                                                                                                                                                   |

```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:226]:
```suggestion
| `CORE_ASPECT_RATIO`   | The core aspect ratio (height / width). This value is ignored if `CORE_UTILIZATION` is undefined.                                                                                                                               |

```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:227]:
```suggestion
| `CORE_MARGIN`         | The margin between the core area and die area, in multiples of SITE heights. The margin is applied to each side. This variable is ignored if `CORE_UTILIZATION` is undefined.                                                         |

```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:228]:
```suggestion
| `DIE_AREA`            | The die area specified as a list of lower-left and upper-right corners in microns (X1 Y1 X2 Y2). This variable is ignored if `CORE_UTILIZATION` and `CORE_ASPECT_RATIO` are defined.                           |

```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:129]: This variable is deprecated and should be removed.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:229]:
```suggestion
| `CORE_AREA`           | The core area specified as a list of lower-left and upper-right corners in microns (X1 Y1 X2 Y2). This variable is ignored if `CORE_UTILIZATION` and `CORE_ASPECT_RATIO` are defined.                                                                 |

```

File: docs/user/GettingStarted.md
Comment (rovinski) [inline docs/user/GettingStarted.md:102]:
```suggestion
Refer to the [Flow variables](./FlowVariables.md) document for details on how to use 
```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:214]: `DESIGN_NICKNAME` is purely for convenience. It doesn't affect synthesis. It just changes the directory name that ORFS outputs to be `DESIGN_NICKNAME` instead of `DESIGN_NAME` in case `DESIGN_NAME` is unwieldy or conflicts with a difference design.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:213]: This is just a hack for ASAP7. It isn't used for any other platform.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:212]: Corner doesn't have to be fast/typical/worst.
```suggestion
| `CORNER`                 | PVT corner library selection.                                                                      |

```

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:211]: This isn't an official flow variable. If it was it's deprecated and not used anywhere.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:190]: Deprecated / unused.

File: docs/user/FlowVariables.md
Comment (rovinski) [inline docs/user/FlowVariables.md:191]: Deprecated / unused

**Review by rovinski — CHANGES_REQUESTED** [review]

**Review by rovinski — APPROVED** [review]
Thanks.

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**
